tests/DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Faker/Fakers/GiasGroupFaker.cs

```diff
@@ -30,6 +33,7 @@ public GiasGroup Generate(TrustToGenerate trustToGenerate, int uid)
         fakeGiasGroup.GroupName = trustToGenerate.Name;
         fakeGiasGroup.GroupType = trustToGenerate.TrustType;
         fakeGiasGroup.GroupUid = uid.ToString();
+        fakeGiasGroup.GroupStatusCode = "OPEN";
```

Comment:
This isn't needed as it's set in the rules

Reply:
Removed